lib/swoosh/adapters/amazon_ses.ex

```diff
@@ -104,12 +104,17 @@ defmodule Swoosh.Adapters.AmazonSES do
   end
 
   defp parse_error_response(body) do
-    node = XMLHelper.parse(body)
+    try do
+      node = XMLHelper.parse(body)
 
-    code = XMLHelper.first_text(node, "//Error/Code")
-    message = XMLHelper.first_text(node, "//Message")
+      code = XMLHelper.first_text(node, "//Error/Code")
+      message = XMLHelper.first_text(node, "//Message")
 
-    %{code: code, message: message}
+      %{code: code, message: message}
+    catch
+      # In case of malformed XML erlang is not throwing error, but is exiting instead
+      :exit, error -> %{code: 400, message: error, malformed_body: body}
```

Comment:
what about:
```
    rescue
      exception ->
         Logger.error(...)
         reraise exception, __STACKTRACE__
```
I would do this to keep the behaviour the same and just add logging. I guess your option is also fine but you likely then have to change the service to actually log these out and make sure we don't suppress the errors